rotkehlchen/api/v1/schemas.py

```diff
@@ -762,6 +762,27 @@ class BaseEvmEventSchema(Schema):
         extra_data = fields.Dict(load_default=None)
         location = LocationField(required=True, limit_to=EVM_EVMLIKE_LOCATIONS)
 
+        @staticmethod
+        def validate_tx_hash(tx_hash: str) -> None:
+            """Check if the provided tx_hash is present in the db.
+            Check evm_events_info rather than evm_transactions here for two reasons:
+            * In the app, a user cannot remove the last event from an evm transaction. So if a
+               tx_hash is in evm_transactions it will also be in evm_events_info at least once.
```

Comment:
my only reason for suggesting `evm_transactions` is because `evm_events_info.tx_hash` does not have a foreign key relationship to `evm_transactions`.

the reasoning you gave makes sense so you can leave as is.